README.rst

```diff
@@ -56,14 +56,23 @@ or in batch mode on Windows, Linux, and even Mac OS.
 
 Documentation and issues
 ------------------------
-For more information, see the `Documentation <https://mapdl.docs.pyansys.com>`_ page.
-For some examples, see the `Examples gallery <https://mapdl.docs.pyansys.com/version/stable/examples/index.html>`_.
+Documentation for the latest stable release of PyMAPDL is hosted at `PyMechanical documentation
+<https://mapdl.docs.pyansys.com/version/stable/>`_.
 
-Feel free to post issues and other questions at `PyMAPDL Issues
-<https://github.com/pyansys/pymapdl/issues>`_.  This is the best place
-to post questions and code.
+In the upper right corner of the documentation's title bar, there is an option for switching from
+viewing the documentation for the latest stable release to viewing the documentation for the
+development version or previously released versions.
 
+You can also `view <https://cheatsheets.docs.pyansys.com/pymapdl_cheat_sheet.png>`_ or
+`download <https://cheatsheets.docs.pyansys.com/pymapdl_cheat_sheet.pdf>`_ the
+PyMAPDL cheat sheet. This one-page reference provides syntax rules and commands
+for using PyMAPDL. 
 
+On the `PyMAPDL Issues <https://github.com/ansys/pymapdl/issues>`_ page,
+you can create issues to submit questions, report bugs, and request new features.
+This is the best place to post questions and code.
```

Comment:
I would mention that for questions, we have PyMAPDL Discussions or the Developer portal.

Reply:
@germa89 This is a good discussion. I have a PR in place for PyMechanical (hence the error you caught in my copy and paste). Thank you! I will change both PRs to indicate that questions should be posted on the PyMAPDL Discussions page or the Developer portal. What about the posting of code? Should that go with the Discussions and Dev portal? I've always been a bit uncomfortable with that add.

Reply:
Posting the code? I guess if the code is part of the question, they better keep everything in the same place (PyMAPDL discussions or Ansys Dev).

If you are proposing a change (with code) or even requesting a change in the code, it should be through a github issue.